.github/workflows/validate-template-workflow.yml

```diff
@@ -92,10 +92,19 @@ jobs:
           npm v ${{ inputs.npm-package }} dist.tarball | xargs curl | tar -xz
           node src/check-npm-package-metadata.js $GITHUB_WORKSPACE/package
           echo "STATUS=success" >> $GITHUB_OUTPUT;
+  check-npm-package-step-requirements:
+    name: Check if we can move to the "check-npm-package" step
+    runs-on: ubuntu-latest
+    needs: [check-github-link, check-npm-package-metadata]
+    steps:
+      - name: Check statuses of previous steps
+        id: check-previous-steps
+        run: |
+          if [ '${{ needs.check-github-link.outputs.status }}' != 'success' ] || [ '${{ needs.check-npm-package-metadata.outputs.status }}' != 'success' ]; then exit 1; fi;
```

Comment:
ℹ️ We check form inputs (github link and npm package name) in parallel to improve a developer experience. We move to the next step only if the supplied form inputs are valid.